app/services/workflow_find_service.rb

```diff
@@ -26,8 +26,6 @@ def find_by_tag_resources(tag_resources)
   end
 
   def fq_tag_names(tag_attrs)
-    GetRemoteTags.new(tag_attrs).process.tags.collect do |tag|
-      "/#{tag[:namespace]}/#{tag[:name]}=#{tag[:value]}"
-    end
+    GetRemoteTags.new(tag_attrs).process.tags.collect { |tag| tag[:tag] }
```

Comment:
You sure you need an another level of calling `tag` attribute? It's already called in `GetRemoteTags.new(tag_attrs).process.tags`